### DISCUSSION TITLE
remote: Your account is suspended.

### DISCUSSION THREAD

**Discussion by AirtonYefri** [community/community#81479] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

good morning.
I already have 4 suspended accounts, to date I have not received a response from Github, when I send my request what is the error and the reason for the suspension.
I previously assumed it was because I had the credentials within my project.
Now I use the credentials manually but even so, they have suspended my 4th account created, without any notice and I want to recover my account and be told that I ended up failing to comply and how I can correct it.
![Screenshot_5](https://github.com/community/community/assets/95237903/a396bb34-de37-44b4-b852-4ad7b84f4627)
suspended accounts
https://github.com/luispurizaca
https://github.com/luispurizacam
https://github.com/negocialab
https://github.com/negocialat
Ticket ID: 2494250 - accounts : negocialat

**Comment by queenofcorgis** [comment, marked as answer]
I took a look into this and it appears our team responded with instructions on how to regain access. But to futureproof this discussion: 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
